python-sdk/nuscenes/eval/detection/README.md

```diff
@@ -193,13 +193,43 @@ To enable a fair comparison between methods, the user will be able to filter the
  
 We define three such filters here which correspond to the tracks in the nuScenes detection challenge.
 Methods will be compared within these tracks and the winners will be decided for each track separately.
-
-* **Lidar detection track**: 
-This track allows only lidar sensor data as input.
-No external data or map data is allowed. The only exception is that ImageNet may be used for pre-training (initialization).
-* **Vision detection track**: 
-This track allows only camera sensor data (images) as input.
-No external data or map data is allowed. The only exception is that ImageNet may be used for pre-training (initialization).
-* **Open detection track**: 
-This is where users can go wild.
-We allow any combination of sensors, map and external data as long as these are reported. 
+Furthermore, there will also be an award for novel ideas, as well as the best student submission.
+
+**Lidar detection track**: 
+* Only lidar input allowed.
+* No external data or map data allowed.
+* May use pre-training.
+  
+**Vision detection track**: 
+* Only camera input allowed.
+* No external data or map data is allowed.
+* May use pre-training.
+ 
+**Open detection track**: 
+* Any sensor input allowed.
+* External data and map data allowed.  
+* May use pre-training.
+
+**Details**:
+* *Sensor input:*
+For the lidar and vision detection tracks we restrict the type of sensor input that may be used.
+Note that this restriction applies only at test time.
+At training time any sensor input may be used.
+In particular this also means that at training time you are allowed to filter the GT boxes using `num_lidar_pts` and `num_radar_pts`, regardless of the track.
```

Comment:
Suggestion to add: `However, during testing, the predicted boxes may _not_ be filtered based on input from other sensor modalities.`